std/hash/_wasm/hmac_hash.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+import init, { HmacSha256Hash, HmacSha512Hash, source } from "./wasm.js";
+
+import type { HmacHasher, Message } from "../hmac_hasher.ts";
+
+await init(source);
+
+type HmacHashMethods = HmacSha256Hash | HmacSha512Hash;
+
+export class HmacHash implements HmacHasher {
```

Comment:
Already generating a wrapper class with wasm_bindgen, wrapping the wrapper class seems like one level of indirection too many.